src/lecture/ten/simmetricDifference/SymmetricDifference.java

```diff
@@ -0,0 +1,17 @@
+package lecture.ten.simmetricDifference;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class SymmetricDifference {
+    public static <T> Set<T> symmetricDifference(Set<? extends T> set1, Set<? extends T> set2) {
+        Set commonness = new HashSet();
```

Comment:
we can simplify this method:
```suggestion
        Set commonness = new HashSet(set1);
```